docs/templates_system/article_media.rst

```diff
@@ -50,3 +50,17 @@ Usage:
         {% endif %}
     {% endgimmelist %}
 
+Get selected rendition only:
+
+
+.. code-block:: twig
+
+    {% gimmelist media from articleMedia with {'article': gimme.article} %}
```

Comment:
Nope, as in publisher we don't have Article renditions. Renditions are per article media (type image). This will can be used like that when we will add support for Superdesk featured image. It will can look like that then:

```
{% gimme rendition with { 'name': '16-9', 'source': 'featuredImage' } %}
    <img src="{{ url(gimme.rendition) }}" style="width: {{ gimme.rendition.width }}px; height: {{ gimme.rendition.height }}px;" />
{% endgimme %}
```

Then we could check for featured image in context current article.